src/UserService.Mappers/UserCredentialsMapper.cs

```diff
@@ -13,18 +13,23 @@ namespace LT.DigitalOffice.UserService.Mappers
     /// </summary>
     public class UserCredentialsMapper : IMapper<UserRequest, DbUserCredentials>
     {
+        internal static string SALT3 = "LT.DigitalOffice.SALT3";
```

Comment:
Maybe move it to the static class and with static method generation password hash?

Reply:
And this field should be const string